deploy/template.yaml

```diff
@@ -707,6 +707,8 @@ Resources:
                   - !If [IsDev02, !Sub "https://${Environment}.02.dev.identity.account.gov.uk", !Sub "https://identity.${Environment}.account.gov.uk"]
             - Name: SESSION_SECRET
               Value: "no-secret"
+            - Name: LANGUAGE_TOGGLE
+              Value: "0"
```

Comment:
for existing env vars we use false/true - could we switch to that instead of 0/1 ?